dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -133,6 +143,28 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::Unregiste
 }
 CATCH_RETURN()
 
+STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::AddToastRegistrationMapping(_In_ PCWSTR processName, _In_ PCWSTR appId) noexcept try
+{
+    auto lock{ m_lock.lock_exclusive() };
+    THROW_HR_IF(WPN_E_PLATFORM_UNAVAILABLE, m_shutdown);
+
+    m_toastRegistrationManager->Add(processName, appId);
```

Comment:
Please add a check to ensure that the longrunning sink has the process before calling these APIs